### PR TITLE
refactor: Remove non-batching event code path (v2 events endpoint)

### PR DESCRIPTION
**Pull request by rmi22186**
## Instructions
 1. PR target branch should be against `development`
 2. PR title name should follow this format: https://github.com/mParticle/mparticle-workflows/blob/main/.github/workflows/pr-title-check.yml
 3. PR branch prefix should follow this format: https://github.com/mParticle/mparticle-workflows/blob/main/.github/workflows/pr-branch-check-name.yml

 ## Summary
Now that all tests have been refactored to use the V3 events endpoint instead of the V2 events endpoint, we can remove all references to the V2 events endpoint from the source code.  

`test/src/tests-temp-session-bug-fix.js` has been removed because V2 is being removed - There was [a bug](https://github.com/mParticle/mparticle-web-sdk/pull/360) that was fixed in September that introduced these duplicated tests in order to test the v3 endpoint - the tests they were duplicated from were still testing the V2 endpoing.  Now that ALL tests have been moved to V3, these are 100% duplicated tests and can be removed.

 ## Testing Plan
 - [X] Was this tested locally? If not, explain why.
 - {explain how this has been tested, and what, if any, additional testing should be done}

 ## Reference Issue (For mParticle employees only.  Ignore if you are an outside contributor)
 - Closes https://go.mparticle.com/work/SQDSDKS-5361

### CODE REVIEW

File: test/integrations/requirejs/test-requirejs.js [hunk 1]
```diff
@@ -2,6 +2,17 @@ describe('Require.JS Pages', function() {
     it('loads mParticle properly', function() {
         var server = new MockHttpServer();
         server.start();
+        window.fetchMock.post('https://jssdks.mparticle.com/v3/JS/test_key/events', 200);
```
Comment (alexs-mparticle) [inline test/integrations/requirejs/test-requirejs.js:5]: Shouldn't this be using the `urls.events` variable?

File: test/src/tests-temp-session-bug-fix.js [hunk 2]
```diff
@@ -1,306 +0,0 @@
-import Utils from './utils';
```
Comment (alexs-mparticle) [inline test/src/tests-temp-session-bug-fix.js:1]: 🎉

File: test/integrations/requirejs/test-requirejs.js (hunk 1 above)
Comment (rmi22186) [reply test/integrations/requirejs/test-requirejs.js:5]: Yes, but I cant link to an expanded resolved comment, so I'll paste a screenshot here:
image

Comment (alexs-mparticle) [reply test/integrations/requirejs/test-requirejs.js:5]: Can you include a link to the Sus Eng ticket as a todo?

**Review by alexs-mparticle — APPROVED** [review]

File: test/integrations/requirejs/test-requirejs.js (hunk 1 above)
Comment (rmi22186) [reply test/integrations/requirejs/test-requirejs.js:6]: done!

**Review by alexs-mparticle — APPROVED** [review]

**Pull request merged**